packages/api/src/services/create_user.ts

```diff
@@ -85,6 +86,9 @@ export const createUser = async (input: {
           group: invite.group,
         })
       }
+
+      await createDefaultFiltersForUser(t)(user.id)
```

Comment:
I suppose this could also be done in postgres with a transaction

Reply:
I mean trigger.

Reply:
OK, I don't really have a preference here. So I think this approach works. The one thing I'd say is it should probably be wrapped in a try/catch. I wouldn't want the user creation to fail.